src/Maker/MakeEntity.php

```diff
@@ -163,9 +163,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             'Entity\\'
         );
 
-        $mappingDriver = $this->doctrineHelper->getMappingDriverForNamespace($entityClassDetails->getFullName());
-
-        if ($mappingDriver instanceof AttributeDriver && !$this->doctrineHelper->isDoctrineSupportingAttributes()) {
+        if (!$this->doctrineHelper->isDoctrineSupportingAttributes()) {
```

Comment:
This would always throw an exception if PHP 8 attributes are supported, right?

We also need `&& $this->doctrineHelper-> doesClassUsesAttributes($entityClassDetails->getFullName())`

Reply:
good catch